scripts/ledger.lic

```diff
@@ -54,6 +56,27 @@ unless failed_to_load.empty?
   exit
 end
 
+
+WITHDRAW_SILVER_REGEX = [
+  /I have a (?<debt>bill) of (?<silver>[\d,]+) silvers? (?:presented by your creditors|that I suggest you pay immediately)/,
+  /Very well, a withdrawal of (?<silver>[\d,]+) silver/,
+  /teller scribbles the transaction into a book and hands you (?<silver>[\d,]+) silver/,
+  /teller carefully records the transaction, (?:and then )?hands you (?<silver>[\d,]+) silver/,
+  /scrip for (?<silver>[\d,]+) silvers?, with a 0 silver fee for the scrip/, # Terras note withdraw, will be moved to notes regex next release

```

Comment:
This regex is the most changes, used to match x,xxx silver fee but as all fees are 0 now this simplifies things. Also does not really fit here as this regex is for silvers, will be replaced by below WITHDRAW_NOTES_REGEX but left here to prevent regressions before that.

Reply:
These should exist inside the module, not outside it. Could use @withdraw_notes_regex as the name as an example and reference it accordingly. That keeps things scoped into the Ledger namespace.

Reply:
Also you can prevent having to use Regexp.union in the matching lines and instead define the variables with it instead. so it becomes:
```ruby
@withdraw_silver_regex = Regexp.union(
  /I have a (?<debt>bill) of (?<silver>[\d,]+) silvers? (?:presented by your creditors|that I suggest you pay immediately)/,
  /Very well, a withdrawal of (?<silver>[\d,]+) silver/,
  /teller scribbles the transaction into a book and hands you (?<silver>[\d,]+) silver/,
  /teller carefully records the transaction, (?:and then )?hands you (?<silver>[\d,]+) silver/,
  /scrip for (?<silver>[\d,]+) silvers?, with a 0 silver fee for the scrip/, # Terras note withdraw, will be moved to notes regex next release
)
```